src/screens/drive/SharedScreen/SharedScreen.tsx

```diff
@@ -66,33 +68,37 @@ export const SharedScreen: React.FC<SharedScreenProps> = ({
     }
 
     if (sharedLinksToRender.length > 0) {
-      return sharedLinksToRender.map((sharedLink, i) => {
+      return sharedLinksToRender.map((sharedLink, i: React.Key) => {
+        logger.info({ sharedLinkData: sharedLink });
```

Comment:
Add some context to the log, if not, this is just a wild object in the console/log file